### PR TITLE
Add correct reference to 3DH and editorial changes

### PR DESCRIPTION
**Pull request by claucece**

### CODE REVIEW

**Comment by claucece** [comment]
cc./ @chris-wood

File: draft-irtf-cfrg-opaque.md [hunk 1]
```diff
@@ -339,8 +339,8 @@ OPAKE was taken.
 OPAQUE relies on the following cryptographic protocols and primitives:
 
 - Oblivious Pseudorandom Function (OPRF, {{!I-D.irtf-cfrg-voprf}}, version -06):
-  - Blind(x): Convert input `x` into an element of the OPRF group, randomize it
-    by some scalar `r`, producing `M`, and output (`r`, `M`).
+  - Blind(x): Convert input `x` into an element of the OPRF group, multiply it
+    with some random scalar `r`, producing `M`, and output (`r`, `M`).
```
Comment (chris-wood) [inline draft-irtf-cfrg-opaque.md:343]: We wrote this intentionally as randomize as that does not assume additive or multiplicative blinding -- can we please revert?

**Comment by chris-wood** [comment]
LGTM pending one revert -- thanks!

File: draft-irtf-cfrg-opaque.md (hunk 1 above)
Comment (claucece) [reply draft-irtf-cfrg-opaque.md]: Done ;)

**Review by chris-wood — APPROVED** [review]

**Pull request merged**